client/components/detailed_profile.js

```diff
@@ -68,14 +72,19 @@ class DetailedProfile extends Component {
 
   render() {
     const { voteSmartBio } = this.props;
+    const { contributors } = this.props;
     if (!voteSmartBio){
       return <div>Loading...</div>;
     }
+    if (!contributors) {
```

Comment:
Feels like this should be part of the component, so that other components can load without waiting for the contribution data.